src/cli.js

```diff
@@ -61,7 +61,12 @@ const read = file => new Promise(resolve => fs.readFile(file, 'utf8', (error, da
 }));
 
 const getPlugins = config => Object.keys(config.plugins || {})
-  .map(plugin => require(plugin)(config.plugins[plugin]));
+  .map(plugin => {
+    let fn = require(plugin);
+    fn = fn && fn.__esModule ? fn.default : fn;
```

Comment:
It might be better to create a wrapper ?
```js
const wrapRequire = fn => fn && fn.__esModule ? fn.default : fn;
```
and call like

```js
  .map(plugin => wrapRequire(require(plugin))(config.plugins[plugin]));
```

Reply:
I named it `interopRequire` instead of `wrapRequire` – that’s how Babel names this function (but with underscore prefix).